cli.js

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env node
+var fs           = require('fs')
+var path         = require('path')
+var pull         = require('pull-stream')
+var toPull       = require('stream-to-pull-stream')
+var File         = require('pull-file')
+var explain      = require('explain-error')
+var Config       = require('ssb-config/inject')
+var Client       = require('ssb-client')
+var createHash   = require('multiblob/util').createHash
+var minimist     = require('minimist')
+var muxrpcli     = require('muxrpcli')
+var cmdAliases   = require('./lib/cli-cmd-aliases')
+//var ProgressBar  = require('./lib/progress')
+var packageJson  = require('./package.json')
+
+//get config as cli options after --, options before that are
+//options to the command.
+
+module.exports = function (config, argv) {
```

Comment:
can we name this function `cli` or something?

as down at the bottom, we call `module.exports(...)` and took me a few moments to grok what was happening.